curriculum/challenges/english/01-responsive-web-design/css-balance-sheet/step-032.md

```diff
@@ -0,0 +1,141 @@
+---
+id: 6193f85ab1de4356da678090
+title: Step 32
+challengeType: 0
+dashedName: step-32
+---
+
+# --description--
+
+Now you need to align the text content of your sheet so that it appears in columns. Start with a `.row` selector to target all of your elements which have the balance values in them. Give this selector a `display` property set to `flex`, and a `justify-content` property set to `flex-end`.
```

Comment:
```suggestion
Now you need to align the text content of your sheet so that it appears in columns. Start with a `.row` selector to target all of your elements that have the balance values in them. Give this selector a `display` property set to `flex`, and a `justify-content` property set to `flex-end`.
```